odl/solvers/advanced/proximal_operators.py

```diff
@@ -568,6 +588,33 @@ def proximal_cconj_l2(space, lam=1, g=None):
 
     Notes
     -----
+    The functional :math:`F` is given by
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_2
```

Comment:
Use `\| x - g \|` to typeset the norm. It doesn't impair the (lower priority) plain text readability significantly but improves the typeset formula.
Search-and-replace across the board.